documentation/sphinx/source/api-common.rst.inc

```diff
@@ -326,6 +326,10 @@
 
     If this option has been set more times with this database than the disable option, snapshot reads will *not* see the effects of prior writes in the same transaction. Disabling this option is equivalent to calling |snapshot-ryw-disable-transaction-option| on each transaction created by this database.
 
+.. |option-db-tr-transaction-logging-max-field-length-blurb| replace::
+
+    Sets the maximum escaped length of key and value fields to be logged to the trace file via the LOG_TRANSACTION option. This is equivalent to calling |transaction-logging-max-field-length-transaction-option| on each transaction created by this database.
```

Comment:
Is there a reason this is `LOG_TRANSACTION` instead of linking to the option? Do we not have that option documented?

Reply:
Yeah, I don't think it's documented and I was deferring that work to https://github.com/apple/foundationdb/issues/1843

Reply:
Yeah, sounds good.